waiter/config-full.edn

```diff
@@ -151,9 +151,9 @@
                         :kind :anonymous
```

Comment:
Maybe we should match cook and have the `:kind` be `:one-user` (https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/components.clj#L316)